packages/datatrak-web/src/Routes.tsx

```diff
@@ -95,10 +95,12 @@ export const Routes = () => {
             </Route>
             <Route path={ROUTES.SURVEY} element={<SurveyPage />}>
               <Route index element={<SurveyStartRedirect />} />
-              <Route path={ROUTES.SURVEY_REVIEW} element={<SurveyReviewScreen />} />
               <Route path={ROUTES.SURVEY_SUCCESS} element={<SurveySuccessScreen />} />
-              <Route path={ROUTES.SURVEY_SCREEN} element={<SurveyScreen />} />
-              <Route path={ROUTES.QUESTIONS} element={<SurveyQuestionsPage />} />
+              <Route element={<ScrollableLayout />}>
+                <Route path={ROUTES.SURVEY_REVIEW} element={<SurveyReviewScreen />} />
+                <Route path={ROUTES.QUESTIONS} element={<SurveyQuestionsPage />} />
```

Comment:
Can you please delete the SurveyQuestionsPage as I just added that for development and testing for Wim and Chao and it's not needed any more.